app/Controllers/Http/MatchController.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const Match = use('App/Models/Match');
+const User = use('App/Models/User');
+const Time = use('App/Models/Time');
+
+class MatchController {
+
+  async index ({ response }) {
+    response.json(await Match.all());
+  }
+
+  async store ({ request, response }) {
+    const { user1_id, user2_id, time_id } = request.all();
+    await Match.create({ user1_id, user2_id, time_id });
+
+    const user1 = await User.findOrFail(user1_id);
```

Comment:
The three findOrFail are okay, but maybe would be a good idea to use a validator for this in order to check if the users and time exists in the database, here an example about that https://adonisjs.com/docs/4.1/validator#_custom_rules

Also, we already have a validator, so you can create one for the match 
Example: https://github.com/DarioNahuel/slack-generate-matches/blob/master/app/Validators/StoreUser.js